CONFLICTS.md

```diff
@@ -99,6 +99,12 @@ In this section we keep track of the reasons, why some restrictions were added t
 
   References: https://github.com/PolishSymfonyCommunity/SymfonyMockerContainer/issues/20
 
+- `doctrine/annotations:^2.0`:
```

Comment:
As the conflict is added only to UiBundle, this note should be also added to that bundle.